src/MyParcelComApi.php

```diff
@@ -300,11 +299,11 @@ public function getServices(ShipmentInterface $shipment = null, array $filters =
         $url->addQuery($this->arrayToFilters([
             'address_from' => [
                 'country_code' => $shipment->getSenderAddress()->getCountryCode(),
-                'region_code'  => $shipment->getSenderAddress()->getRegionCode(),
+                'postal_code'  => $shipment->getSenderAddress()->getPostalCode(),
             ],
             'address_to'   => [
                 'country_code' => $shipment->getRecipientAddress()->getCountryCode(),
-                'region_code'  => $shipment->getRecipientAddress()->getRegionCode(),
+                'postal_code'  => $shipment->getRecipientAddress()->getPostalCode(),
```

Comment:
Are we sure that this is backwards-compatible with parties already using the SDK?

Reply:
They cannot omit the `postal_code`, since it's a required field. This function receives the full `$shipment` which is assumed to contain the address, not only a country code and change later.